test/TestServerProjects/body-formdata-urlencoded/Generated/Models/PostContentSchemaPetType.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace body_formdata_urlencoded.Models
+{
+    /// <summary> Can take a value of dog, or cat, or fish. </summary>
+    public readonly partial struct PostContentSchemaPetType : IEquatable<PostContentSchemaPetType>
```

Comment:
Both types got modeled as extensible. I guess we need to put modelAsString false on another one.

Reply:
https://github.com/Azure/autorest.testserver/pull/278****